data/json/items/armor/ballistic_armor.json

```
@@ -95,6 +95,143 @@
+          { "type": "plastic_pad", "covered_by_mat": 100, "thickness": 2.0 }
+        ],
+        "coverage": 100,
+        "cover_vitals": 100,
+        "covers": [ "torso", "arm_l", "arm_r" ],
+        "encumbrance": [ 71, 80 ]
+      }
+    ]
+  },
+  {
+    "id": "jacket_eod_light",
+    "type": "TOOL_ARMOR",
+    "category": "armor",
+    "name": { "str": "tactical EOD jacket" },
+    "description": "An armored jacket constructed from Kevlar and Nomex for use by explosive ordnance disposal technicians in 'non-permissive environments'.  It is lighter than normal EOD armor to allow for wearing a plate carrier beneath it.  It features two pockets in the arms covering the deltoids for additional optional protection.",
+    "weight": "7000 g",
+    "volume": "15 L",
+    "price": 410160,
+    "price_postapoc": 5900,
+    "warmth": 40,
+    "material": [ "kevlar", "plastic_pad", "nomex" ],
+    "symbol": "[",
+    "color": "light_gray",
+    "environmental_protection": 2,
+    "material_thickness": 7,
+    "flags": [ "STURDY", "OUTER", "RAINPROOF", "ONLY_ONE" ],
+    "use_action": [ { "type": "attach_molle", "size": 15 }, { "type": "detach_molle" } ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "ablative": true,
+        "volume_encumber_modifier": 0,
+        "max_contains_volume": "3000 ml",
+        "max_contains_weight": "7 kg",
+        "max_item_length": "46 cm",
+        "moves": 1100,
+        "description": "Pocket for a huge armor plate.",
+        "flag_restriction": [ "ABLATIVE_LARGE" ]
+      },
+      {
+        "pocket_type": "CONTAINER",
+        "ablative": true,
+        "volume_encumber_modifier": 0,
+        "max_contains_volume": "1558 ml",
+        "max_contains_weight": "1600 kg",
+        "max_item_length": "21 cm",
+        "min_item_length": "21 cm",
+        "//": "min. length really just a hack to avoid adding a flag.",
+        "moves": 800,
+        "description": "Pockets for deltoid armor plates.",
+        "flag_restriction": [ "ABLATIVE_MEDIUM" ]
+      }
+    ],
+    "armor": [
+      {
+        "material": [
+          { "type": "nylon", "covered_by_mat": 100, "thickness": 1.0 },
+          { "type": "kevlar", "covered_by_mat": 100, "thickness": 4.4 },
+          { "type": "plastic_pad", "covered_by_mat": 100, "thickness": 2 }
+        ],
+        "coverage": 100,
+        "cover_vitals": 100,
+        "covers": [ "torso", "arm_l", "arm_r" ],
+        "encumbrance": [ 31, 55 ]
+      }
+    ]
+  },
+  {
+    "id": "eod_plate",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "EOD fragmentation plate" },
+    "description": "A polygonal fragmentation plate with a slightly concave profile, made of thin layers of steel, UHMW polyethylene, foam, and kevlar.  Though certified to provide protection from fragmentation, it will not provide much protection from firearms.  This is intended to be inserted into an EOD suit, and can't be worn on its own.",
```

```suggestion
    "description": "A polygonal fragmentation plate with a slightly concave profile, made of thin layers of steel, UHMW polyethylene, foam, and Kevlar.  Though certified to provide protection from fragmentation, it will not provide much protection from firearms.  This is intended to be inserted into an EOD suit, and can't be worn on its own.",
```
*Angry DuPont noises*